test/shared/src/main/scala/zio/test/render/TestRenderer.scala

```diff
@@ -132,6 +146,13 @@ trait TestRenderer {
     )
   }
 
+  def renderOutput(output: Chunk[String]): Message =
+    Message(
+      Line.fromString("          Output Produced by Test         ".red.underlined, 2) +:
+        output.map(s => Line.fromString("| ".red + s.yellow, 2)) :+
+        Line.fromString("==========================================\n".red, 2)
```

Comment:
I would prefer this to be in the ConsoleRenderer instead.

Reply:
Totally! Done.